tests/Integration/build.sh

```diff
@@ -1,12 +1,12 @@
 #!/usr/bin/env bash
 
 APP_FOLDER=test-app
-SYMFONY_VERSION=3.3
+SYMFONY_VERSION=$1
```

Comment:
Could we pass a default value for this as we are not informing that this is required?